README.md

```diff
@@ -16,10 +16,20 @@ You’ll need a Kubernetes cluster to run against. You can use [KIND](https://si
 ### To run a kind cluster locally 
 
 ```sh
-make cluster-up CLUSTER_PROVIDER='kind' CI_DEPLOY=true
+make cluster-up
```

Comment:
Since `local-dev-cluster` also supports microshift I think it's best to have `CLUSTER_PROVIDER` env variable